posthog/models/user.py

```diff
@@ -151,4 +151,32 @@ def leave(self, *, organization: Organization) -> None:
                 )
                 self.save()
 
+    def get_analytics_metadata(self):
+
+        team_member_count_all: int = OrganizationMembership.objects.filter(
+            organization__in=self.organizations.all(),
+        ).values("user_id").distinct().count()
+
+        return {
+            "realm": "cloud" if getattr(settings, "MULTI_TENANCY", False) else "hosted",
+            "ee_available": settings.EE_AVAILABLE,
```

Comment:
Let's use `is_ee_available()` here as it has slightly different and more accurate semantics.